app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Gathering school direct contact details from higher education institutions
+description: Using a spreadsheet to help higher education institutions invite their school direct partners to use  Manage
+date: 2021-04-08
+---
+
+We’ve been looking at ways to help higher education institutions (HEIs) to:
+
+- set themselves up to use Manage
+- invite their school direct partners (SDs) to set themselves up
+
+We started by designing a [small online service that allowed HEIs to set themselves up and invite SDs](https://bat-design-history.netlify.app/manage-teacher-training-applications/self-service-registration/). 
+
+After testing that design we decided to explore a simpler approach, focusing on gathering contact details for SDs.
+
+## What we did and why
+
+We created a spreadsheet template that we could send to HEIs as a CSV file. The columns in the template were:
+
+- partner (pre-filled with the name of an SD) 
+- code (pre-filled with the SD’s provider code)
+- contact name (to be completed by the HEI user)
+- contact email (to be completed by the HEI user)
+
+We decided to keep the headings short as we have no control over column width in a CSV file. When we tested it using Microsoft Excel, the default column width was very narrow.
+
+If an SD was already using Manage, the contact name and email cells would be pre-filled with ‘Not needed - already has account’. 
+
+We also created an email explaining what we wanted users to do. It asked them to either: 
+
+- download the spreadsheet, complete it and email it back
+- fill in a Google Sheets version of the spreadsheet online
+
+The email said that courses would become available as soon as the contact at the SD set up their account.
+
+Since the previous iteration, we’d discovered that HEI users have to sign a data sharing agreement (DSA) in order to get access to the ‘sandbox’ version of Manage. They access this so that they can experiment before signing up to use the service with real data.
```

Comment:
What happens if a user bypasses the sandbox process and goes straight to the full service? In that instance, they'd need to sign a DSA.